pkg/docker/config/config.go

```diff
@@ -54,10 +54,18 @@ var (
 
 // SetCredentials stores the username and password in a location
 // appropriate for sys and the users’ configuration.
+// An error will be part of the returned multierror if the provided key is
+// namespaced and a credentials helper other than the AuthenticationFileHelper
+// is used.
```

Comment:
This doesn’t tell the caller anything actionable (beyond “on error the `error` return value is set”, which is not typically worth writing out), and right now, as is, can just be dropped.

OTOH the function should document the accepted formats for `key` (and that might be a good place to mention that sub-registry keys might not be supported, depending on users’ configuration).

Reply:
Yep, added docs about the key and dropped the comment on the error.